crates/raito-cairo-args/src/serializer.rs

```diff
@@ -0,0 +1,509 @@
+use crate::error::CairoSerializeError;
```

Comment:
Can we partially reuse https://github.com/starkware-libs/stwo-cairo/tree/main/stwo_cairo_prover/crates/cairo-serialize here?

Reply:
What exactly would you like to reuse?